addons/languages/thai/pack/src/main/res/xml/thai_keyboards.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Keyboards>
+    <Keyboard nameResId="@string/thai_keyboard" iconResId="@drawable/ic_status_thai"
```

Comment:
Can you attach two screenshots of these keyboards?

Reply:
Pics:

![2](https://user-images.githubusercontent.com/49518157/86807369-00465400-c0a4-11ea-8d57-ecca1346f7ba.jpg)

![1](https://user-images.githubusercontent.com/49518157/86807365-ff152700-c0a3-11ea-8dbd-b1ef78183274.jpg)

Reply:
I messed up the second one a bit. Pic is the fixed version, pushing the change now.

Reply:
thanks